js.js

```diff
@@ -0,0 +1,124 @@
+console.log("hello world!");
+console.log("hallo wereld!");
+console.log("selam dunya!");
+
+console.log("I'm awesome");
+console.log('I\'m awesome');
+
+
+let x;
+console.log("the value of x is undefined");
+console.log(x);
+x = 5;
+console.log("the value of x is 5");
+console.log(x);
+
+
+let y = "string";
+console.log("the value of y is string");
+console.log(y);
+y = "new string";
+console.log("the value of y now is 'new string'");
+console.log(y);
+
+
+let z = 7.25;
+console.log(z);
+let a = Math.floor(z);
+console.log(a);
+let h;
+if (z>a){
```

Comment:
If you add some spaces here, it becomes easier to read:

```
if (z > a) {
```

Reply:
thanx .I'll pay more attention for this details